docs/administration/necessary-adjustments.md

```diff
@@ -0,0 +1,29 @@
+# Necessary adjustments
```

Comment:
What about `Custom Funding Case Types`?

Reply:
They are mentioned in line 11, but I don't go further into detail because I am missing the necessary knowledge and also the time. In the future there could be a more elaborative explanation at this page.